app/config/config_test.yml

```diff
@@ -14,3 +14,19 @@ web_profiler:
 
 swiftmailer:
     disable_delivery: true
+
+# It's recommended to use a separate database for tests. This allows to have a
+# fixed and known set of data fixtures, it simplifies the code of tests and it
+# makes them more robust.
+# In this case we just define a different path for the application database,
+# but you can even use an entirely database engine for tests.
```

Comment:
using a different database engine should not be recommended IMO: some of your queries might depend on the engine being used, and would not work on the other.
Even if you rely on Doctrine only and never write SQL yourselves, it may have platform-specific bugs (especially on platforms being less tested like SqlServer or Oracle), for instance when paginating complex queries (there is a huge work in the ORM currently to make the pagination actually work on complex queries). And some features are simply not supported on some platforms